keps/sig-multicluster/2149-clusterid/README.md

```diff
@@ -353,6 +363,38 @@ Contains a unique identifier for the containing cluster.
 
 **Reusing cluster names**: Since an `id.k8s.io ClusterProperty` has no restrictions on whether or not a ClusterProperty can be repeatable, if a cluster unregisters from a ClusterSet it is permitted under this standard to rejoin later with the same `id.k8s.io ClusterProperty` it had before. Similarly, a *different* cluster could join a ClusterSet with the same `id.k8s.io ClusterProperty` that had been used by another cluster previously, as long as both do not have membership in the same ClusterSet at the same time. Finally, two or more clusters may have the same `id.k8s.io ClusterProperty` concurrently (though they **should** not; see "Uniqueness" above) *as long as* they both do not have membership in the same ClusterSet.
 
+#### Property: `network.k8s.io`
+
+Contains an identifier representing the network for the cluster.
+
+
+##### Uniqueness
+
+*   The identifier **may not** exist (as its only applicable for multi-network scenario) and **need not** be unique
+
+##### Lifespan
+
+*   The identifier, if exists, **should** be immutable for the lifespan of a ClusterSet membership.
```

Comment:
Why is this linked to ClusterSet at all?  Can't I have this for standaalone clusters?

Reply:
Change of a network would mean the cluster has to renew its membership in the multi-cluster's ClusterSet. This statement conveys that changing so would need a cluster to unsubscribe and resubscribe its ClusterSet membership. The property can be used outside of multi-cluster scenarios as needed.

Reply:
I don't know that this is true in either direction.

First, I don't really know what it would mean to change the network on a running cluster, but assuming thsat was a thing that was possible and sensible, I don't know anything about the clusterset implementation that would let me make statements about the implications of suchs a change.

Second, I feel like I should be able to asser this property even if I am not in a clusterset.